tooling/build/scripts/seed-from-repo/.env.template

```diff
@@ -0,0 +1,3 @@
+DATABASE_URL=postgresql://root:root@localhost/isomer_studio
```

Comment:
for consistency purposes with `.env.example`, should we use `postgres://root:root@localhost:5432/app`?